### PR TITLE
DD-1280: mapping tests + bug fixes

### PR DESCRIPTION
**Pull request by jo-pol**
Fixes DD-1280: mapping tests + bug fixes

# Description of changes

* new test that sword example `all-rules` indeed produces (most) metadata fields
  for now it reads a stable raw file from github
* new tests for metadata-blocks `dansArchaeologyMetadata`, `dansTemporalSpatial` and `dansRelationMetadata`
* fixed attribute name space issue for AbrAcquisitionMethod (AR005)
* fixed trimming for abr report number (AR004) and other MultiplePrimitive fields
* fixed `<afm:keyvaluepair>` (xpath for localname only like for `<afm:othmat_codebook>`)
* fixed empty series field

# How to test

# Related PRs

(Add links)

*

# Notify

@DANS-KNAW/dataversedans

### CODE REVIEW

File: src/test/java/nl/knaw/dans/ingest/core/service/mapper/MappingTestHelper.java [hunk 1]
```diff
@@ -26,37 +33,81 @@
...
+
+    private static final IngestFlowConfig config = getIngestFlowConfig();
+
+    private static IngestFlowConfig getIngestFlowConfig() {
+        IngestFlowConfig config;
+        try {
+            config = new YamlConfigurationFactory<>(DdIngestFlowConfiguration.class, Validators.newValidator(), Jackson.newObjectMapper(), "dw")
+                .build(FileInputStream::new, "src/test/resources/debug-etc/config.yml")
+                .getIngestFlow();
+            config.setMappingDefsDir(Paths.get("src/test/resources/debug-etc"));
```
Comment (jo-pol) [inline src/test/java/nl/knaw/dans/ingest/core/service/mapper/MappingTestHelper.java:86]: Would `src/main/assembly/dist/cfg` be a better option?

File: src/main/java/nl/knaw/dans/ingest/core/service/mapper/mapping/FileElement.java [hunk 2]
```diff
@@ -170,7 +170,7 @@ static Map<String, List<String>> getKeyValuePairs(Node node, String filename, St
     }
 
     static String replaceForbiddenCharactersInPath(String dirPath) {
-        if (dirPath == null) {
+        if (dirPath == null || dirPath.isBlank()) {
```
Comment (jo-pol) [inline src/main/java/nl/knaw/dans/ingest/core/service/mapper/mapping/FileElement.java:173]: Before this change the github build failed on `SwordExamplesTest`, after this change `FileElementTest` fails.

Comment (janvanmansum) [reply src/main/java/nl/knaw/dans/ingest/core/service/mapper/mapping/FileElement.java]: I think StringUtils.isBlank will do this without the `||`.

Comment (jo-pol) [reply src/main/java/nl/knaw/dans/ingest/core/service/mapper/mapping/FileElement.java]: Outdated code, it was reverted.

**Pull request merged**